wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -383,17 +383,18 @@ of [Cardano](https://github.com/input-output-hk/cardano-sl).
 ### Request format (all versions)
 
 Issuing requests against this API is conceptually not very different from any other web service out there. The API
-is versioned, meaning is possible to access different versions of the latter by changing the _version number_ in the URL.
+is **versioned**, meaning is possible to access different versions of the latter by changing the _version number_ in the URL.
```

Comment:
"meaning it's possible"
"latter by changing" -> "new api by adding the"

Reply:
good catch!